### ISSUE TITLE
Doesn't sync .env variables

### ISSUE THREAD

**Issue by mbalandis** [dongido001/vscode-sync-env#3]
```
PUSHER_APP_ID=12345
PUSHER_APP_KEY=abcs
PUSHER_APP_SECRET=aaaaa
PUSHER_APP_CLUSTER=eu

MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"

```
doesn't sync the bottom 2

**Comment by dongido001** [comment]
Fix in #5, will be available on next release.

**Comment by dongido001** [comment]
Available now on version 1.0.4
